### PR TITLE
feat: Requestrr

### PR DESCRIPTION
**Pull request by flying-sausages**
## Description
Add Requestrr to box installs (Replacing #599)

## Fixes issues: 
Fixes no issues.

## Proposed Changes:
- Add Requestrr to the stack

## Change Categories

- New feature / Application 

## Checklist

- [x] Docs have been made OR are not necessary
    - PR link: https://github.com/liaralabs/docs.swizzin.ltd/pull/56
- [x] Changes to panel have been made OR are not necessary
    - PR link: https://github.com/liaralabs/swizzin_dashboard/pull/32
- [x] Code is formatted [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Shellcheck isn't screaming [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Prints to terminal are handled [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#printing-into-the-terminal)
- [x] I have commented my code, particularly in hard-to-understand areas

## Test scenarios

Has been tested on AMD64, should also be respectful of ARM.

### Architectures

|   			| `amd64` 	| `armhf` 	| `arm64` 	| Unspecified 	|
|--------		|-------- 	|-------- 	|-------- 	|----------		|
| Focal 		|			|			|	✅		|				|
| Bionic		|			|			|			|				|
| Buster		|	✅		|			|			|				|
| Stretch		|			|			|			|				|
| Raspbian  	|			|			|			|				|

### 🛠🛠 TODO
- [x] Docs (@HarleyL7)
- [x] Updater script
- [x] Removal script
- [x] Panel stuffs
- [x] Testing
- [x] Auto-config?
- [ ] Unattended login with swizz credentials

### ❌❌ Currently failing
Tested. Nginx + Installs + removals succeed.

Continues to serve on all interfaces when nginx is enabled, although 400's on all requests to the port; except those via the reverse-proxy.

### CODE REVIEW

File: sources/functions/requestrr [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      "Hostname": "",
+      "Port": 5055,
+      "ApiKey": "",
+      "DefaultApiUserID": "",
+      "UseSSL": false,
+      "Version": "1"
+    },
+    "Radarr": {
+      "Hostname": "${radarr[0]}",
+      "Port": ${radarr[1]},
```
Comment (flying-sausages) [inline sources/functions/requestrr:92]: This value needs to have a default, when radarr is not installed, otherwise the JSON is invalid.

Comment (flying-sausages) [reply sources/functions/requestrr:92]:
```json
    "Overseerr": {
      "Hostname": "",
      "Port": 5055,
      "ApiKey": "",
      "DefaultApiUserID": "",
      "UseSSL": false,
      "Version": "1"
    },
    "Radarr": {
      "Hostname": "",
      "Port": , // <<< This is what ya gets, and that ain't valid
      "ApiKey": "",
      "BaseUrl": "",
      "MovieProfileId": "1",
      "MovieRootFolder": "",
      "MovieMinimumAvailability": "",
      "MovieTags": [],
      "AnimeProfileId": "1",
```

Comment (brettpetch) [reply sources/functions/requestrr:92]: Ah. Forgot to set it as an else 8989 default.

Comment (flying-sausages) [reply sources/functions/requestrr:92]: Could use this
```bash
FOO="${VARIABLE:-default}"  # If variable not set or null, use "default".
```

File: sources/functions/requestrr [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    "Command": "movie"
+  },
+  "TvShows": {
+    "Client": "Disabled",
+    "Command": "tv",
+    "Restrictions": "None"
+  },
+  "Port": 4545,
+  "BaseUrl" : "/requestrr",
+  "Version": "1.12.0"
```
Comment (flying-sausages) [inline sources/functions/requestrr:142]: Holup, won't this change? This should probably not be here?

Comment (brettpetch) [reply sources/functions/requestrr:142]: Devs seem to have written an upgrade path for the config file in the past

**Comment by flying-sausages** [comment]
For track record:
- Arr owners will be retrievable after #719
- auth will be done by basic auth after upstream dev adds the option for it, so we won't need to hash things

**Review by brettpetch — COMMENTED** [review]
I think that this should be good now.

**Comment by liaralabs** [comment]
Requestrr is now abandoned.

**Pull request closed without merging**